src/insns/load_16bit_fp_sp.adoc

```diff
@@ -27,12 +27,6 @@ Standard floating point load instructions, authorised by the capability in <<ddc
 NOTE: These instructions are available in RV32 {cheri_int_mode_name} only.
 In {cheri_cap_mode_name} they are remapped to <<C.LC>>/<<C.LCSP>>.
 
-NOTE: In {cheri_int_mode_name}, these instructions may be remapped to
```

Comment:
This is still true? But deleting it seems fine.

Reply:
it's true but as Zilsd support is now fully in I thought I'd remove it